compiler/rustc_interface/src/interface.rs

```diff
@@ -329,6 +329,8 @@ pub fn create_compiler_and_run<R>(config: Config, f: impl FnOnce(&Compiler) -> R
     })
 }
 
+// JUSTIFICATION: interface fn
```

Comment:
Apologies, I'll improve this comment. Just that I would expect the functions in `rustc_interface` to access some options directly because they might be running before the `Session` is constructed, or something along those lines.